frontend/src/app/search-city.service.ts

```diff
@@ -13,19 +13,20 @@ export class SearchCityService {
 
   /**
    * Returns the first 10 cities matching a request
-   * @param term
-   * @returns {any}
    */
   search(term: string): Observable<Array<CityModel>> {
     if (term === '') {
       return Observable.of([]);
     }
 
-    const vicopoUrl = `${window.location.protocol}//vicopo.selfbuild.fr/cherche/${term}`;
-
-    return this.http
-      .get<any>(vicopoUrl)
-      .map(response => response.cities.slice(0, 10) as Array<CityModel>);
+    return this.http.get('/api/cities', { params : new HttpParams().set('query', term)});
```

Comment:
Not sure about it, but I think we handled the calls to vicopo in a specific fashion to avoid to send the Authorization header. It should not be necessary anymore, so some parts of `JwtInterceptorService` may be removed (and associated tests).